libevmasm/Assembly.h

```diff
@@ -171,11 +179,25 @@ class Assembly
 
 	unsigned codeSize(unsigned subTagSize) const;
 
+	/// Add all assembly items from given JSON array. This function imports the items by iterating through
+	/// the code array. This method only works on clean Assembly objects that don't have any items defined yet.
+	/// @param _json JSON array that contains assembly items (e.g. json['.code'])
+	/// @param _sourceList list of source names.
+	void importAssemblyItemsFromJSON(Json::Value const& _code, std::vector<std::string> const& _sourceList);
```

Comment:
```suggestion
	void importAssemblyItemsFromJSON(Json::Value const& _json, std::vector<std::string> const& _sourceList);
```
just to match the documentation above.